libctru/source/services/mvd.c

```diff
@@ -9,27 +9,27 @@
 #include <3ds/svc.h>
 #include <3ds/srv.h>
 #include <3ds/os.h>
-#include <3ds/linear.h>
+#include <3ds/allocator/linear.h>
 #include <3ds/synchronization.h>
 #include <3ds/services/mvd.h>
 #include <3ds/ipc.h>
 
 Handle mvdstdHandle;
 static int mvdstdRefCount;
-static mvdstdMode mvdstd_mode;
-static mvdstdTypeInput mvdstd_input_type;
-static mvdstdTypeOutput mvdstd_output_type;
+static MVDSTD_Mode mvdstd_mode;
+static MVDSTD_InputFormat mvdstd_input_type;
+static MVDSTD_OutputFormat mvdstd_output_type;
 static u32 *mvdstd_workbuf;
 static size_t mvdstd_workbufsize;
 
-static Result mvdstdipc_Initialize(u32 *buf, u32 bufsize, Handle kprocess)
+static Result MVDSTD_Initialize(u32* buf, u32 bufsize)
 {
 	u32* cmdbuf = getThreadCommandBuffer();
 	cmdbuf[0] = IPC_MakeHeader(0x1,2,2); // 0x10082
 	cmdbuf[1] = (u32)buf;
 	cmdbuf[2] = bufsize;
 	cmdbuf[3] = IPC_Desc_SharedHandles(1);
-	cmdbuf[4] = kprocess;
+	cmdbuf[4] = CUR_PROCESS_HANDLE;
```

Comment:
I don't know if it *has* to be, but I assume its similar to others like GSPGPU_FlushDataCache. No idea what purpose any other values would have, nor what they would even do.